### DISCUSSION TITLE
AWS Bedrock Claude 3 Integration

### DISCUSSION THREAD

**Discussion by AdityaIndoori** [Pythagora-io/gpt-pilot#824] [Q&A]
Hey all, 

Does anyone have documentation on how to setup GPT Pilot with AWS Bedrock and use Claude 3? I couldn't find anything in the Readme or related docs. I just found a comment in the .env file saying: 

> You only need to set this if not using Anthropic API directly (eg. via proxy or AWS Bedrock)
> ANTHROPIC_ENDPOINT= 

But nothing more. So was wondering if anyone knew how. 

Thanks!

**Comment by phalexo** [comment]
I got the impression that Bedrock still uses Anthrop\c's API. You have to enable Anthropic within Bedrock. There are videos on Youtube that cover Bedrock's LLM models, and briefly discuss Anthropic.
